src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -229,20 +241,36 @@ def update_site_configs(resource_group_name, name, slot=None,
                         remote_debugging_enabled=None, web_sockets_enabled=None,  # pylint: disable=unused-argument
                         always_on=None, auto_heal_enabled=None,  # pylint: disable=unused-argument
                         use32_bit_worker_process=None,  # pylint: disable=unused-argument
+                        arr_affinity=None,  # pylint: disable=unused-argument
                         app_command_line=None):  # pylint: disable=unused-argument
+
     configs = get_site_configs(resource_group_name, name, slot)
+    site_config = configs.site_config
+    print("Config is {}".format(site_config))
```

Comment:
Why is this a raw print statement instead of a logging statement?  At a minimum it should print to STDERR, not STDOUT.

Reply:
Consider using logger.warning if you want it to be visible to the user.